ydb/mvp/core/core_ydb.h

```diff
@@ -305,7 +305,7 @@ struct TYdbLocation {
 
     std::unique_ptr<NYdb::NDataStreams::V1::TDataStreamsClient> GetDataStreamsClientPtr(TStringBuf endpoint, TStringBuf scheme, const NYdb::TCommonClientSettings& settings = NYdb::TCommonClientSettings()) const;
 
-    NYdb::NTable::TTableClient GetTableClient(const TRequest& request, const NYdb::NTable::TClientSettings& defaultClientSettings = {}) const;
+    NYdb::NTable::TTableClient GetTableClient(const TRequest& request, const NYdb::NTable::TClientSettings& defaultClientSettings, const TString& metaDatabaseTokenName = "") const;
```

Comment:
why it's meta database token name here? are we always connect table client to "meta" database?

Reply:
changed to a less specific name `tokenName`